front/lib-svelte/src/demande-aide-mon-aide-cyber/FormulaireDemandeAide.svelte

```diff
@@ -0,0 +1,231 @@
+<script lang="ts">
+  import Bouton from '../ui/Bouton.svelte';
+  import ChampTexte from '../ui/ChampTexte.svelte';
+  import SelectionOrganisation from '../ui/formulaire/SelectionOrganisation.svelte';
+  import ControleFormulaire from '../ui/ControleFormulaire.svelte';
+  import Formulaire from '../ui/Formulaire.svelte';
+  import type { OrganisationDisponible } from '../ui/formulaire/SelectionOrganisation.types';
+  import { createEventDispatcher } from 'svelte';
+  import type { DonneesFormulaireDemandeAide } from './DonneesFormulaireDemandeAide';
+  import { validationChamp } from '../directives/validationChamp';
+  import Alerte from '../ui/Alerte.svelte';
+
+  let formulaire: Formulaire;
+  export let enCoursEnvoi: boolean;
+  export let formulaireSoumis: boolean;
+
+  export let erreurs: string;
+
+  let entite: OrganisationDisponible;
+  let email: string;
+  let estEnRelationAvecUnUtilisateur: boolean;
```

Comment:
```suggestion
  let estDejaEnContact: boolean;
```
je trouve ça plus clair personnellement